lib/tasks/emails.rake

```diff
@@ -0,0 +1,20 @@
+namespace :metamaps do
+  desc "delivers recent map activity digest emails to users"
+  task deliver_map_activity_emails: :environment do
+    summarize_map_activity
+  end
+
+  def summarize_map_activity
+    Map.all.find_each do |map|
+      map.followers.each do |user|
+        # add logging and rescue-ing
+        # and a notification of failure
+        next unless MapPolicy.new(user, map).show? # just in case the permission changed
+        next unless user.emails_allowed
```

Comment:
nice, these two lines are v clever

are you going to update the email permissioning in this PR? It seems like the right time to do it. I could maybe push some commits to the branch if it helps